addons/packages/ako-operator/1.6.0/bundle/config/kapp-config.yaml

```diff
@@ -23,3 +23,15 @@ data:
       sources: [existing, new]
       resourceMatchers:
       - apiVersionKindMatcher: {apiVersion: networking.tkg.tanzu.vmware.com/v1alpha1, kind: AKODeploymentConfig}
+    - paths:
+      - [data, username]
+      - [data, password]
+      type: copy
+      sources: [existing, new]
+      resourceMatchers:
+      - kindNamespaceNameMatcher: {kind: Secret, namespace: tkg-system-networking, name: avi-controller-credentials}
```

Comment:
This is the default name, you can use YTT overlay to get the input from `ako-operator` add-on secret